projects/packages/jitm/src/class-jitm.php

```diff
@@ -80,14 +80,7 @@ public function register() {
 	 * @param \WP_Screen $screen WP Core's screen object.
 	 */
 	public function prepare_jitms( $screen ) {
-		if ( ! in_array(
-			$screen->id,
-			array(
-				'jetpack_page_akismet-key-config',
-				'admin_page_jetpack_modules',
-			),
-			true
-		) ) {
+		if ( apply_filters( 'jetpack_display_jitms', true, $screen->id ) ) {
```

Comment:
Could you add a docblock here? Given that we also have the existing `jetpack_just_in_time_msgs` filter, part of me wonders if we shouldn't rename this filter to be more verbose? Maybe `jetpack_display_jitms_on_screen`? What do you think?

Reply:
Works for me. I added a clarifying comment since this function already has a docblock, but please let me know if you meant something else.

Reply:
We need a docblock for the filter itself, like [this](https://github.com/Automattic/jetpack/blob/master/projects/packages/jitm/src/class-jitm.php#L52). The requirements for filter docblocks are described in the WordPress Coding Standards [here](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).

I think the filter info from the docblock is used on [developer.jetpack.com](developer.jetpack.com).